sdk/src/main/java/org/coralibre/android/sdk/fakegms/nearby/exposurenotification/ExposureConfiguration.java

```diff
@@ -227,12 +272,12 @@ public int getTransmissionRiskScore(final int transmissionRiskValue) {
 
 
     /**
-     * @param attenuationValue the bluetooth attenuation value from an exposure, in dB:
-     *                         {@code transmission power - } {@link CapturedData#getRssi()}
+     * @param attenuationValue           the bluetooth attenuation value from an exposure, in dB:
+     *                                   {@code transmission power - } {@link CapturedData#getRssi()}
      * @param daysSinceLastExposureValue the number of days since the last exposure
-     * @param durationValue the duration of an exposure, in minutes
-     * @param transmissionRiskValue the user defined risk value associated to an exposure,
-     *                              must be >= 0 and <= 7
+     * @param durationValue              the duration of an exposure, in minutes
+     * @param transmissionRiskValue      the user defined risk value associated to an exposure,
+     *                                   must be >= 0 and <= 7
```

Comment:
Why adding those spaces? Did the IDE do it automatically? In my humble opinion it looked better as it was before, though feel free to ignore me ;-)